app/views/content/guidance/financial-support-for-international-applicants.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: Financial support for international applicants
+draft: true
```

Comment:
Does this need to be draft still? I see we've published it now @peteryates

Reply:
I marked it as draft because we don't really want it to appear in the sitemap, it only exists for people who've bookmarked the page (or have the URL in an email or something)